.github/workflows/build-linux.yml

```diff
@@ -45,12 +67,7 @@ jobs:
         sleep 3
         herbstluftwm &
         sleep 1
-        python -m pytest -sv
-
-    - name: Test MacOS
-      if: matrix.os == 'macos-latest'
-      run: |
-        python -m pytest -sv
+        python -m pytest -sv -k "not ert3"
```

Comment:
Could you give some context here?

Reply:
Ert3 is now running against ert-storage. Ert-storage needs a pgsql instance, and services on github actions are only available for linux workers.

Reply:
I mean, as a comment in the code